tools/osx/macosx_bundle.sh

```diff
@@ -111,13 +112,13 @@ install -d  "${RESOURCES}" \
 
 msg "Copying release files:"
 ditto "${CMAKE_BUILD_TYPE}/MacOS" "${MACOS}"
-ditto "${CMAKE_BUILD_TYPE}/Resources" "${RESOURCES}"
+ditto "Resources" "${RESOURCES}"
 
 msg "Copying dependencies from ${GTK_PREFIX}:"
 CheckLink "${EXECUTABLE}"
 
 msg "Copying library modules from ${GTK_PREFIX}:"
-ditto --arch "${arch}" {"${GTK_PREFIX}/lib","${LIB}"}/gdk-pixbuf-2.0
+ditto --arch "${arch}" {"${GDK_PREFIX}/lib","${LIB}"}/gdk-pixbuf-2.0
```

Comment:
@Krijger what do the outer curly braces do?
**{**"${GDK_PREFIX}/lib","${LIB}"**}**/gdk-pixbuf-2.0

Reply:
That was new for me as well, but it's Brace Expansion https://www.linuxjournal.com/content/bash-brace-expansion

```
echo {1,2}/test
1/test 2/test

echo 1,2/test
1,2/test
```

Reply:
Personally, I evolved to prefer explicit and simple script code to fancy expansion if the latter is unnecessary, as it's just more readable and easier for others (and myself 6 months down the line) to grasp and fix:
```
ditto --arch "${arch}" \
    "${GDK_PREFIX}/lib/gdk-pixbuf-2.0" \
    "${LIB}/gdk-pixbuf-2.0"
```

Reply:
I'm fine with that, but my aim was to be as least intrusive to the original code as possible. I think the goal of the original author may have been to keep things as DRY as possible.

Do you want me to change it in the scope of this PR?

Reply:
@krijger not necessary in the scope of this PR.